pkg/ipmi/ipmi.go

```diff
@@ -254,17 +254,7 @@ func (i *IPMI) RawSendRecv(msg Msg) ([]byte, error) {
 	}
 
 	if err := ioctlSetReq(i.Fd(), _IPMICTL_SEND_COMMAND, req); err != nil {
```

Comment:
repeat IPMICTL_SEND_COMMAND if err == syscall.EINTR (it may equally get disrupted.)

Reply:
Should there be a timeout on this too then?

Reply:
nah, you can remove the timeout here -- it'll eventually work.